docs/monthly/2023-02-monthly.md

```diff
@@ -0,0 +1,185 @@
+---
+title: February 2023
+slug: 2023-02
+authors: ch1bo
+tags: [monthly]
+---
+
+This report summarizes the work on Hydra since January 2023. It serves as
+preparation for the monthly review meeting, where the teams update major project
+stakeholders on recent developments to gather their feedback on the proposed
+plan forward.
+
+## Roadmap
+
+Looking at the [roadmap](https://github.com/orgs/input-output-hk/projects/21),
+there are a few things that changed this month:
+
+- In general, there will be quite a lot of items in flux as we align the
+  technical roadmap to support our high-level objectives for the year.
+
+  + New items for example: [create a use case centric
+  roadmap #710](https://github.com/input-output-hk/hydra/issues/710) and [open
+  specification with change process
+  #693](https://github.com/input-output-hk/hydra/issues/693).
+
+- Also note, that we have not had a release in a while and we want to have
+  smaller faster releasable items again
+
+- [Aligning implementation with HeadV1 specification
+  #452](https://github.com/input-output-hk/hydra/issues/452) was more work than
+  anticipated. There was quite some forth and back with the writing the
+  specification and discussions with researchers leading to scope creep.
+  + Most-important: made on-chain behavior consistent with specification
+  + Follow-ups for some off-chain protocol logic:
+    - [Authenticate network messages #727](https://github.com/input-output-hk/hydra/issues/727)
+    - [ReqSn only sends transaction ids #728](https://github.com/input-output-hk/hydra/issues/728)
+  
+- Version **0.9.0** will be the first release which is developed against a full technical specification. We expect a release in early March.
+
+- Version **0.10.0** will be the first mainnet compatible release. For which we created and groomed the main feature item [Mainnnet compatibility #713](https://github.com/input-output-hk/hydra/issues/713):
+  - Not only making it technically compatible, but also add guard rails to limit
+    risk (“not shoot yourself in the foot .. with a real gun now”) and document
+    known issues.
+
+- For a fully maintained version **1.0.0** we still need to address some things, but we aim for an increased rate of planned releases in the next months as we march toward it.
+  
+![](./img/2023-02-roadmap.png) <small><center> The latest roadmap with multiple new items and starting to reflect Hydra objectives of 2023.</center></small>
+
+## Development
+
+[Issues and pull requests closed since last
+report](https://github.com/input-output-hk/hydra/issues?q=is%3Aclosed+sort%3Aupdated-desc+closed%3A2023-01-24..2023-02-28)
```

Comment:
Suggestion for next MR: Lighten up the ad hoc writing about devs and let the issues "speak for themselves", this will alleviate your workload significantly and will "force" people to provide better issues/PRs, perhaps?